### ISSUE TITLE
Ask about struct bind

### ISSUE THREAD

**Issue by Dup4** [bytedance/sonic-cpp#46]
<img width="379" alt="image" src="https://user-images.githubusercontent.com/38343778/211147637-9222d720-59d5-46a7-ac1b-3857ea113d78.png">

- Is the `struct bind` capability in roadmap a convenient way to serialize and deserialize a struct?
- The plan is which bi-monthly will support this?

**Comment by liuq19** [comment]
yes. bind is serde between JSON and a c++ type, including a struct.  And there has been a POC demo for binding and it will open source in the future.  But there is no specific time for it.
